preprocess.py

```diff
@@ -7,7 +7,8 @@
 
 # new max length of our sentences
 # before was 512
-maxLength = 32
+# space for cls and sep
+maxLength = 32 - 2
```

Comment:
lowkey we should put all of the hyperparameters in hyperParam.py so that we can easily see and tweak as needed- not urgent tho ofc